dotcom-rendering/src/web/components/MostViewedRight.test.tsx

```diff
@@ -1,14 +1,14 @@
 import { render } from '@testing-library/react';
 
-import { useApi as useApi_ } from '@root/src/web/lib/useApi';
+import { useApi as useApi_ } from '../lib/useApi';
```

Comment:
Is there a reason to switch from absolute to relative imports?

Reply:
There are various benefits to relative paths as well as cons. It's a whole trade offs thing. But I think in most cases it's just easier to allow VSCode to automatically add them. I'm pretty sure this is the reason everyone keeps doing this